src/components/bookList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Book from './book';
+
+const BooksList = () => {
+  const bookList = [
+    {
+      id: 1,
+    },
+    {
+      id: 2,
+    },
+  ];
```

Comment:
[ Optional Changes ]

- [ ] A book is expected to have a title and an author and then when you loop through the array, you pass a single book as a prop to the Book component which will be received as props inside the Book component and render the data to it. Kindly add the title and author for each of your objects in the bookList array. 👍 

```js
 const bookList = [
    {
      id: 1,
      title: "The way of Kings",
      author: "Brandon"
    },
    {
      id: 2,
      title: "The dragon reborn",
      author: "Marcus Rufus"
    },
  ];
```